Makefile

```diff
@@ -1,11 +1,10 @@
 # Variables
```

Comment:
I don't think the Makefile or update-deps.sh are useful. Let's just remove them and update the README.

Reply:
Nuked from orbit. It's the only way to be sure...